src/detail-page/components/TabsContainer.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import { Card, Tabs } from '@material-ui/core';
+import styled from 'styled-components';
+
+import { TabPanel } from './TabPanel';
+import { StyledTab } from './StyledTab';
+import { JobsTab } from './JobsTab';
+import { LearnTab } from './LearnTab';
+import { ResourcesTab } from './ResourcesTab';
+import { categories } from './TabCategories';
+
+const StyledTabs = styled(Tabs)`
+    border-bottom: 1px solid ${props => { return props.theme.palette.text.disabled; }};
+    color: inherit;
+`;
+
+const StyledCard = styled(Card)`
+width: 100%;
+`;
+
+/* For now TabContainer receive props from its parent container
+with what to learn, jobs and resources.
+Because it isn't determined how they will look like it's type is unknown.
+It should be change when it will be determined */
+interface TabProps {
+    learn: unknown;
+    jobs: unknown;
+    resources: unknown;
+  }
+
+export const TabsContainer: React.FC<TabProps> = ({
+  learn,
+  jobs,
+  resources,
+}) => {
+  const [choosesCategory, setChoosesCategory] = useState(0);
+
+  const TabsHeading = categories.map((category: any, id: number) => {
```

Comment:
We shouldn't set `category` type as `any` - it's actually `Category` :) we can set it if we export `Category` type from `TabCategories.tsx`. But we actually don't have to set a type by ourselves - TypeScript is clever enough to infer the type from the value we're passing (`categories` array) :) https://www.typescriptlang.org/docs/handbook/type-inference.html
On an unrelated issue: we should rename `id` to `index` to reflect more accurately what this variable is.

Reply:
Ok. So as I understand I don't need to set any types here? Because TypeScript will do it by itself?

Reply:
Exactly! We don't need to provide types for params of the callback function passed to `map`, TypeScript will know what types `category` and `id` because of the types we set for `categories` :)